app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -144,10 +145,10 @@ fun RiskText(
   Text(
       modifier = Modifier
           .offset {
-            IntOffset(
-                x = clampedOffsetX.toInt(),
-                y = 0
-            )
+              IntOffset(
+                  x = clampedOffsetX.toInt(),
+                  y = 0
+              )
```

Comment:
Can you check code styling once? Diff shows whitespaces being added here.

Reply:
Fixed